src/xhk.c

```diff
@@ -606,7 +606,7 @@ static void handle_signal(int signum)
         if (LocalScreen.display)
             reattach_device(LocalScreen.display, LocalScreen.keyboard.deviceid, LocalScreen.keyboard.attachment);
 
-        ApplicationRunning = False;
+        ApplicationRunning = false;
```

Comment:
It seems I was using True across the code base too ... I'm not sure how I ended up using "True and False" rather than "true and false" , as  <stdbool.h> is included which defines them as lower case.

The fact that it compiles of course means that some code somewhere is providing values for True and False.

I hope that there is not such a distinction that False != false though ...

Either way I agree we should change both False to false and True to true ...